config/default.js

```diff
@@ -38,7 +40,7 @@ module.exports = {
   },
 
   userToken: {
-    cookieName: 'rds-session',
+    cookieName: `rds-session-${NODE_ENV}`,
```

Comment:
We just need to rename the non-production cookies.

Reply:
@sanyogita14 , can you override the cookie name in production config to be `rds-session`?
We do not want the cookie name in production to be `rds-session-production`.

Reply:
Alright..so I'll just specify it separately for the production file, right?

Reply:
Yes @sanyogita14